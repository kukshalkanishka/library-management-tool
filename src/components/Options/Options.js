import React from "react";
import { Link } from "react-router-dom";

const Options = () => {
  return (
    <div className="form options">
      <h3>FUN TO-DO</h3>
      <Link to="/login" className="option">
        Login
      </Link>
      <Link to="/signup" className="option">
        Sign Up
      </Link>
    </div>
  );
};

export default Options;
