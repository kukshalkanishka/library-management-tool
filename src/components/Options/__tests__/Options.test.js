import React from "react";
import { shallow, configure } from "enzyme";
import Options from "../Options";

import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("Options", () => {
  it("should render properly", () => {
    const wrapper = shallow(<Options />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
