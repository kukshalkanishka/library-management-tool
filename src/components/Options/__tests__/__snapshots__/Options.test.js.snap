// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Options should render properly 1`] = `
"<div className=\\"form options\\">
  <h3>
    FUN TO-DO
  </h3>
  <Link to=\\"/login\\" className=\\"option\\">
    Login
  </Link>
  <Link to=\\"/signup\\" className=\\"option\\">
    Sign Up
  </Link>
</div>"
`;
